nucypher/blockchain/eth/actors.py

```diff
@@ -246,12 +246,13 @@ def deploy_network_contracts(self,
                                  policy_secret: str,
                                  adjudicator_secret: str,
                                  user_escrow_proxy_secret: str,
+                                 gas_limit: int,
                                  ) -> Tuple[dict, dict]:
         """
         Musketeers, if you will; Deploy the "big three" contracts to the blockchain.
         """
 
-        token_txs, token_deployer = self.deploy_contract(contract_name='NuCypherToken')
+        token_txs, token_deployer = self.deploy_contract(contract_name='NuCypherToken', gas_limit=gas_limit)
```

Comment:
Maybe set `gas_limit` to `None` by default? It will help when gas estimation works properly

Reply:
👍